### ISSUE TITLE
Fix error in Calculate Accessibility Matrix tool

### ISSUE THREAD

**Issue by mmorang** [Esri/public-transit-tools#78]
User used the same layer for both Origins and Destinations.

1900-01-03 08:58:00
1900-01-03 08:59:00
1900-01-03 09:00:00
Calculating statistics and writing results...
Failed script CalculateAccessibilityMatrix...

Traceback (most recent call last):
  File "E:\Repositories\CMTA_GIS_Tools\Admin Tools\GTFS_Transit_Tools\AddGTFStoaNetworkDataset_0_5_4_0\scripts\CalculateAccessibility.py", line 213, in <module>
    for dest in OD_count_dict[origin_OID]:
KeyError: 2536

Failed to execute (CalculateAccessibilityMatrix).
